backend/kaos_backend/util/validators.py

```diff
@@ -159,3 +162,21 @@ def wrapper(*args, **kwargs):
         return function(*args, **kwargs)
 
     return wrapper
+
+
+def validate_request(function):
```

Comment:
Could you please rename `validate_request` to a name that expresses the intent of this decorator more? Request validation could be anything from Authorization to Schema checking. How about `authenticate` or `auth_required`?

Here are some of the more popular examples I saw out there:
- [`@login_required` from Flask-Security](https://pythonhosted.org/Flask-Security/api.html)
- [`@auth.login_required` from flask-HTTPAuth](https://flask-httpauth.readthedocs.io/en/latest/)
- [`@authenticate`](https://eddmann.com/posts/using-basic-auth-and-decorators-in-pythons-flask/)

Note that they use the word `login` due to maintenance of Cookies and/or Sessions, for your implementation, you might want to call it differently.